gridt/src/app/core/models/movement.model.ts

```diff
@@ -38,5 +38,15 @@ export interface Movement {
    */
   description?: string;
 
+    /**
+   * A location where the movement is active.
+   */
+  location?: string;
```

Comment:
Could the location be an Array also? I can imagine a movement being in the north of the Netherlands for example.

Reply:
It can become that later on, however for this initial request we agreed on just the provinces. Now from my point of view that meant 1 province. But that can change as a separate issue is made for the location.